src/tests/JIT/Directed/coverage/importer/Desktop/byrefsubbyref1.il

```diff
@@ -18,25 +18,26 @@
 .method public static native int byrefsubbyref(class ctest& V_1, class ctest& V_2)
 {
 ldarg 0
+conv.u
 ldarg 1
-sub
 conv.u
+sub
 ret
 }
 .method public static native int byrefsubi4(class ctest& V_1, int32 V_2)
 {
 ldarg 0
+conv.u
```

Comment:
ldarg 1 also needs `conv.u` doesn't it? to make it 64bit friendly

but since we agreed to fix the jit instead it's better to leave it as is then?